LICENSE.md

```diff
@@ -270,3 +270,16 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ### contextMenu [2.1.1](https://github.com/swisnl/jQuery-contextMenu)
 MIT licensed
+
+--------------------------------------------------------------------------------
+
+### FileSaver [1.1.20160328](https://github.com/eligrey/FileSaver.js/)
+### Blob [2014-07-24](http://purl.eligrey.com/github/Blob.js/blob/master/Blob.js)
```

Comment:
Would you mind breaking this down in two sections, and wrapping the lines to fit 80 columns?

For Blob, instead of linking to the date, can you link to the SHA-1 of the commit where we got this file from? Instead of linking to the file can you link to the GitHub repo?